src/js/jsx/search/SearchBar.jsx

```diff
@@ -45,7 +45,8 @@ define(function (require, exports, module) {
      * @const
      * @type {string}
      */
-    var PLACEHOLDER_ID = "NO_OPTIONS-placeholder";
+    var PLACEHOLDER_ID = "NO_OPTIONS-placeholder",
+        ADOBE_STOCK_FILTER = "ADOBE_STOCK";
```

Comment:
I think I understand but I want to clarify in person with you before I go ahead with this.